src/resources/HostedPages/HostedPages.model.ts

```diff
@@ -0,0 +1,87 @@
+import {Paginated} from '../BaseInterfaces.js';
+
+export interface ListHostedPagesParams extends Paginated {
+    /**
+     * A substring that must appear in a search interface configuration name for this configuration to appear in results.
+     */
+    filter?: string;
+
+    /**
+     * The sort direction of the results.
+     * Possible values:
+     * - `'asc'`: ascending order
+     * - `'desc'`: descending order.
+     */
+    order?: 'desc' | 'asc';
+}
+
+export interface HostedPage {
+    /**
+     * The unique identifier of the hosted page.
+     */
+    id: string;
+    /**
+     * A short descriptive name to help manage hosted pages.
+     */
+    name: string;
+    /**
+     * The HTML markup of the hosted page.
+     */
+    html: string;
+    /**
+     * JavaScript resource appended to the header of a specific hosted page in the target Coveo Cloud organization.
```

Comment:
Since this is an array, I gather there can be many?
```suggestion
     * The JavaScript resources appended to the header of a specific hosted page in the target Coveo Cloud organization.
```